panther_manager/README.md

```diff
@@ -27,45 +27,63 @@ echo $USERNAME 'ALL=(ALL) NOPASSWD: /sbin/poweroff, /sbin/reboot, /sbin/shutdown
-- `~fun_enable_hysteresis` [*float*, default: **60.0**]: minimum time of fan being turned on.
-- `~overwrite_fan_control` [*bool*, default: **false**]: enable the fan to be always on at start of the node. It can be turned off later by `/panther/manager/overwrite_fan_control` service.
-- `~self_identity_file` [*float*, default: **~/.ssh/id_rsa**]: identity file global path to shutdown device running this node. If not, set defaults to `~default_identity_file`.
-- `~self_ip` [*string*, default: **127.0.0.1**]: IP used to shutdown device running this node.
-- `~self_username` [*string*, default: **husarion**]: username used to shutdown device running this node.
 - `~shutdown_timeout` [*float*, default: **15.0**]: time in seconds to wait for graceful shutdown. After timeout power will be cut from all devices.
+- `~update_charging_anim_step` [*float*, default: **0.1**]: percentage value representing a step for updating the charging battery animation.
 
 ### system_status_node.py
 
-Publishes stats status of the internal computer. Stats include CPU utilization and temperature, as well as disc and RAM usage.
+Publishes stats status of the built-in computer. Stats include CPU utilization and temperature, as well as disc and RAM usage.
 
 #### Publishes
 
-- `/panther/system_status` [*panther_msgs/SystemStatus*]: information about internal computer CPU temperature, utilization and disc and RAM usage.
\ No newline at end of file
+- `/panther/system_status` [*panther_msgs/SystemStatus*]: information about internal computer CPU temperature, utilization, disc and RAM usage.
+
+## BehaviorTree 
+
+### Nodes
+
+#### Actions
+
+- `CallSetBoolService` - allows calling standard **std_srvs/SetBool** ROS service.
+- `CallSetLedAnimationService` - allows calling custom type **panther_msgs/SetLEDAnimation** ROS service.
+- `CallTriggerService` - allows calling standard **std_srvs/Trigger** ROS service.
+- `ShutdownHostsFromFile` - allows to shutdown devices based on YAML file. Returns `FAILURE` only when YAML file is incorrect or a path to the file does not exists. If it fails to execute command in a remote host node will proceed tothe next device from the list.
+- `ShutdownSingleHost` - allows to shutdown single device. Will return `SUCCESS` only when device can be reached and the command was executed.
+
+#### Decorators
+
+- `TickAfterTimeout` - will skip child until the specified time has passed. It can be used to specify the frequency at which a node or subtree is triggered.
+
+### Trees
+
+- `Lights` - tree responsible for scheduling animations displayed on LED panels, based on the Husarion Panther robot's system state.
+- `Safety` - tree responsible for monitoring the Panther robot's state and handling safety measures, such as  cooling the robot in case of high CPU or battery temperature.
+- `Shutdown` - tree responsible for gracefull shutdown of robot components, user computers and the built-in computer.
```

Comment:
There is still no information about black board input and output ports and no information about constants like animation IDs and so on. Can you at least make draft to be finished in a separate task?